### PR TITLE
[MTL-2019] Add sponsors and set cfp date

### PR DESCRIPTION
**Pull request by pdion891**
* Add all latest sponsor
* Updated cfp dates and link to submission form
* put some french content
* published event venue

Registration system not ready yet so link still hidden.

Thanks !

### CODE REVIEW

File: data/events/2019-montreal.yml [hunk 1]
```diff
@@ -69,6 +78,9 @@ team_members: # Name is the only required field for team members.
     employer: "eStruxture Data Centers"
   - name: "Carol Trang"
     employer: "CloudOps"
+  - name: "Carine Daoust"
+    github: "cdaoustd"
+    employer: "CloudOps"
```
Comment (bridgetkromhout) [inline data/events/2019-montreal.yml]: Adding an organizer means you need to email info@devopsdays.org with the new organizer's email address so we can add them to Slack and email. Thanks.

File: content/events/2019-montreal/location.md [hunk 2]
```diff
@@ -4,7 +4,20 @@ Type = "event"
 Description = "Location for devopsdays Montreal 2019"
 +++
 
-Watch this space for information about the venue including address, map/direction, parking/transit, and any hotel details.
+La seconde édition de devopsdays Montréal sera a l'[Double Tree Hilton](https://doubletree3.hilton.com/en/hotels/quebec/doubletree-by-hilton-montreal-YMQDTDT/maps-directions/index.html).
```
Comment (cornazano) [inline content/events/2019-montreal/location.md]: Should be "au Doubletree", n'est-ce pas? Also (not as sure on this one): seconde, or deuxième ?

File: content/events/2019-montreal/sponsor.md [hunk 3]
```diff
@@ -3,14 +3,22 @@ Title = "Sponsor"
 Type = "event"
 Description = "Sponsor devopsdays Montreal 2019"
 +++
+Nous apprécions beaucoup les commandiatires de notre évènement. Si vous aimeriez contribuer, <b>lisez les informations <a href="https://objects-qc.cloud.ca/v1/5ef827605f884961b94881e928e7a250/Julia/DevOps Days Sponsorship Deck 2019.pdf" target="_blank">ici</a></b> et envoyez-nous un courriel: [{{< email_organizers >}}].
```
Comment (cornazano) [inline content/events/2019-montreal/sponsor.md]: Small typo: événement (accent on the second "e")

File: data/events/2019-montreal.yml (hunk 1 above)
Comment (pdion891) [reply data/events/2019-montreal.yml]: Thanks for the correction and info, I just sent email to info@ for @cornazano and @cdaoustd.

**Pull request merged**